chapters/1.md

```diff
@@ -10,7 +10,7 @@ Maria is purchasing a car from George, and in this chapter we'll consider three
 
 If Maria sends the money to George through the traditional banking system, they trust three intermediate parties (their respective banks and a payment system which can act as intermediary between the two different banks) to symbolically move the funds for them.
 
-There is no actual movement of physical bills or assets; both banks simply edit their respective databases to show that the funds have been transferred. When Maria submits the transaction to her bank (whether by wire transfer, her bank's website, or an app), the payment system ask to her bank to subtract $2,500 from her account balance on their ledger, then contacts George's bank and requests that they add $2,500 to George's balance.
+There is no actual movement of physical bills or assets; both banks simply edit their respective databases to show that the funds have been transferred. When Maria submits the transaction to her bank (whether by wire transfer, her bank's website, or an app), the payment system asks her bank to subtract $2,500 from her account balance on their ledger, then contacts George's bank and requests that they add $2,500 to George's balance.
```

Comment:
Note: This was already fixed in the second edition: https://github.com/monerobook/monerobook/blob/second-edition/chapters/1.md